mps-model-adapters/src/main/kotlin/org/modelix/model/mpsadapters/MPSModelAsNode.kt

```diff
@@ -28,10 +30,11 @@ import org.modelix.model.api.IReferenceLink
 import org.modelix.model.api.NodeReference
 import org.modelix.model.api.NullChildLink
 import org.modelix.model.area.IArea
-import org.modelix.model.data.NodeData
 
 data class MPSModelAsNode(val model: SModel) : IDeprecatedNodeDefaults {
 
+    private val builtinModel = BuiltinLanguages.MPSRepositoryConcepts.Model
```

Comment:
What's the reason for this member being created here? Could probably be part of the companion at least?